### PR TITLE
Ignore non-CSV files when processing reports

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

Ignore non-CSV files when processing reports.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/FtpClient.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ public void upload(LocalSourceFile file, boolean isSmokeTestDoc) {
 
                 return sftp.ls(configProperties.getReportsFolder())
                     .stream()
-                    .filter(RemoteResourceInfo::isRegularFile)
+                    .filter(f -> f.isRegularFile() && f.getName().toLowerCase().endsWith(".csv"))
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/FtpClient.java]: not sure whether `RemoteResourceInfo` became unused import but ci will pick this up - not a blocker

**Review by doncem — DISMISSED** [review]
:palm_tree:

File: src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/FtpClient.java (hunk 1 above)
Comment (lgonczar) [reply src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/FtpClient.java]: Well spotted. Removed the import.

**Review by doncem — DISMISSED** [review]
:kiwi_fruit:

File: src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/FtpClient.java (hunk 1 above)
Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/FtpClient.java]: how about 2 filters?

**Review by rkondratowicz — APPROVED** [review]

**Pull request merged**